src/uu/ls/src/ls.rs

```diff
@@ -3008,6 +3008,17 @@ fn display_inode(metadata: &Metadata) -> String {
 #[allow(unused_variables)]
 fn get_security_context(config: &Config, p_buf: &Path, must_dereference: bool) -> String {
     let substitute_string = "?".to_string();
+    if must_dereference {
```

Comment:
A small comment explaining that this is for compatibility with GNU would be great

Reply:
@tertsdiepraam : I've added a small comment, awaiting your review :)

Reply:
Nice! There's one crucial bit of info missing from that comment. You could add this to the end: "even when the system does not support SELinux"